compositor_pipeline/src/pipeline/decoder/audio.rs

```diff
@@ -51,13 +35,13 @@ trait AudioDecoderExt {
     fn decoded_sample_rate(&self) -> u32;
 }
 
-pub fn start_audio_decoder_thread(
-    opts: AudioDecoderOptions,
+pub fn start_audio_resampler_only_thread(
```

Comment:
I don't like the word `only` being included in the name of this function.

I spent a while thinking about this and I don't see a good way to make this better.